docker/common.ps1

```diff
@@ -69,11 +69,26 @@ function GetUbuntuImageTags($repo, $version, $arch) {
         
 }
 
-function GetWindowsImageTags($repo, $version) {
-    return @(
-        "$($repo):$($version)-windows",
-        "$($repo):windows-latest",
-        "$($repo):windows-latest-lts",
-        "$($repo):5.4-windows-latest"
-    )
+function GetWindowsImageTags($repo, $version, $WinVer) {
+    switch ($winver) {
+        "1809" {
+            return @(
+                "$($repo):$($version)-windows",
+                "$($repo):windows-latest",
+                "$($repo):windows-latest-lts",
```

Comment:
add 1809 tag please

Reply:
do we want 2022 as `latest`? that would make sense, but it might break people's servers. Maybe we should drop `latest` altogether?

Reply:
I think that the best way is to leave it like this.

We should have the `latest` tag, it feels like a standard, but the kernels across different windows versions are various. 

Many people could have their pipelines set up using this floating docker tag. If they're using a system that isn't working on 2022 kernel, it's gonna break it.

Reply:
but after all, now it looks like it is the one and only opportunity to delete the 'latest' tag

Reply:
I think we should deprecate it - stop pushing new images there. `*latest*` should be gone including `5.4-windows-latest`. Instead we should have
`5.4-windows-lts1809` and `5.4-windows-ltsc2022` and so on. Let's wait for approval from @ppekrol though.
There are also CI/CD challenges ahead here.

Reply:
@poissoncorp Talked with Paweł. List of tags we need:

v5.4 images

for 1809	
	
"$($repo):$($version)-windows-1809",
"$($repo):windows-1809-latest",
"$($repo):windows-1809-latest-lts",
"$($repo):5.4-windows-1809-latest"

for ltsc2022

"$($repo):$($version)-windows-ltsc2022",
"$($repo):windows-ltsc2022-latest",
"$($repo):windows-ltsc2022-latest-lts",
"$($repo):5.4-windows-ltsc2022-latest"

Reply:
removed the latest tags
added tags mentioned above